test/gui/shared/steps/steps.py

```diff
@@ -233,6 +233,13 @@ def step(context, stepPart1, stepPart2):
     )
 
 
+@Then(r"^(.*) on the server", regexp=True)
```

Comment:
I'm not fully sure, but you may make this work with any step with `on the server` in it
```suggestion
@Then(r"^(.*)on the server(.*)", regexp=True)
```

Reply:
This way doesn't seem to work. So, I think we should keep this as it is.